membersrvc/membersrvc.yaml

```diff
@@ -101,8 +101,18 @@ eca:
 
 tca:
           attribute-encryption:
-                 enabled: false
-
+                 enabled: true
+aca:
+          attributes:
+              attribute-entry-0: diego;institution_a;company;IBM;2015-01-01T00:00:00-03:00;;
+              attribute-entry-1: diego;institution_a;position;Software Staff;2015-01-01T00:00:00-03:00;2015-07-12T23:59:59-03:00;
```

Comment:
ACA attributes has been added within the membersrvc.yaml temporarily in the future external attribute authority will be invoked.  Each attribute has the format:

  userid;affiliaition;attributename;attributevalue;validfrom;validto;

validTo empty means that the attribute doesn't expire. That is the reason you see ";;" 
Anyway I'm adding a format comment in this file and modifying the company name to a fantasy name.